.github/workflows/kickstart-tests.yml

```diff
@@ -133,6 +135,15 @@ jobs:
           repository: rhinstaller/kickstart-tests
           path: kickstart-tests
 
+      - name: Generate test cases
+        working-directory: ./kickstart-tests
+        run: scripts/generate-testcases.py -t ./testlib/test_cases/kstest-template.tc.yaml.j2 . -o ./testlib/test_cases
+
+      - name: Clone Permian repositories
+        run: |
+          git clone https://github.com/rhinstaller/permian.git
+          git clone https://github.com/rhinstaller/tclib.git
```

Comment:
Please use checkout action instead of clone.

Reply:
Good point, I'll change it.

Reply:
Done